### DISCUSSION TITLE
[low priority] but must read

### DISCUSSION THREAD

**Discussion by HierarchyMan** [diamond-rip/Eden#434] [Ideas]
So the code is gonna be a big mess if you keep adding multiple spigots support in the base plugin
So make addons for each and every spigot and remove the feature entirely from Eden
Kinda like frost one addon for carbon spigot 
And other for wind and so on

**Comment by GoodestEnglish** [comment]
It will not. Adding more spigot support will not change the base plugin. By what "adding more spigot support" mean, is allow Eden to control the knockback system from the spigot, which I made a separate addon just for this. 

You can find it here: https://github.com/diamond-rip/Eden-SpigotAPI
